scripts/install_plugins.sh

```diff
@@ -5,32 +5,84 @@ CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 source "$CURRENT_DIR/shared_functions.sh"
 
 clone() {
-	local plugin=$1
-	cd $SHARED_TPM_PATH &&
+	local plugin; plugin="$1"
+	cd "$SHARED_TPM_PATH" &&
```

Comment:
I appreciate the quoting of variables (as SHARED_TPM_PATH could indeed contain spaces), but if you'd done so in a separate commit the changes would've been a lot simpler to review ;)